### DISCUSSION TITLE
[Enhancement] Provide the same subscription controls to closable discussions as are provided for issues.

### DISCUSSION THREAD

**Discussion by beedell-roke** [community/community#61069] [Discussions] [answered]
### Select Topic Area

Product Feedback

### Body

I want to be able to subscribe only to issue closure for a discussion, too:

-	#### Discussion

	<img width="312" height="150" alt="Screenshot 2025-09-23 at 17-40-20 _ · RokeJulianLockhart_RokeJulianLockhart · Discussion #15" src="https://github.com/user-attachments/assets/dd8a8ac2-a84c-45d4-8376-91fa12fa91d1" />

-	#### Issue

	1.	<img width="296" height="145" alt="Screenshot 2025-09-23 at 17-40-02 _ · Issue #16 · RokeJulianLockhart_RokeJulianLockhart" src="https://github.com/user-attachments/assets/07cc0c8c-834e-4d92-bf88-4f02d5869a0d" />

	1.	<img width="639" height="689" alt="@{'Date'='20250923T174354+0100'; 'Type'='Photo'; 'Origin'='multiple accounts · Issue #26 · element-hq_element-android — Mozilla Firefox'}#" src="https://github.com/user-attachments/assets/a0735669-4c4c-4dfb-975c-6506b7cd6c95" />

**Comment by beedell-roke** [comment]
News, for those subscribed. GitHub intends to implement a new GUI soon, and as part of that, originally removed this functionality from issues too. Luckily, due to the speed of a beta tester who implored that they reverse the decision, a “message” has been sent to the development team about it. You can read more at [`discussions/132506#discussioncomment-10613144`](https://github.com/orgs/community/discussions/132506#discussioncomment-10613144) — I suggest that you make it known that you disagree with the original decision to remove it.

**Reply by beedell-roke** [reply]
<del>This has been remediated.</del>

**Reply by beedell-roke** [reply, marked as answer]
Actually, per [`discussions/204563#discussioncomment-17966514`](https://github.com/orgs/community/discussions/204563#discussioncomment-17966514), the custom subscription functionality has been removed, again! Consequently, this FR is permanently blocked.

**Comment by LiteBrite82** [comment]
💬 Your Product Feedback Has Been Submitted 🎉
Thank you for taking the time to share your insights with us! Your feedback is invaluable as we build a better GitHub experience for all our users.

Here's what you can expect moving forward ⏩
- Your input will be carefully reviewed and cataloged by members of our product teams. 
- Other users may engage with your post, sharing their own perspectives or experiences. 
- GitHub staff may reach out for further clarification or insight. 

Where to look to see what's shipping 👀
- Read the [Changelog](https://github.blog/changelog/) for real-time updates on the latest GitHub features, enhancements, and calls for feedback.
- Explore our [Product Roadmap](https://github.com/orgs/github/projects/4247), which details upcoming major releases and initiatives.

What you can do in the meantime 💻
- Upvote and comment on other user feedback Discussions that resonate with you.
- Add more information at any point! Useful details include: use cases, relevant labels, desired outcomes, and any accompanying screenshots.

As a member of the GitHub community, your participation is essential. While we can't promise that every suggestion will be implemented, we want to emphasize that your feedback is instrumental in guiding our decisions and priorities.

Thank you once again for your contribution to making GitHub even better! We're grateful for your ongoing support and collaboration in shaping the future of our platform. ⭐
